homeassistant/components/honeywell/config_flow.py

```diff
@@ -0,0 +1,50 @@
+"""Config flow to configure the honeywell integration."""
+import somecomfort
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN
+
+
+class HoneywellConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a honeywell config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Create config entry. Show the setup form to the user."""
+        errors = {}
+
+        if user_input is not None:
+            valid = await self.is_valid(user_input["username"], user_input["password"])
```

Comment:
Why can you use `CONF_USERNAME` and `CONF_PASSWORD` here?

Or even (you'll need to check this):
```python
valid = await self.is_valid(**user_input)
```

Reply:
...later down, you may need:
```python
async def is_valid(self, username, password, **kwargs)
```

NOte this is converting kwargs to positional, so you may want to do something different.

Reply:
Holy man this `**kwargs` concept is slick! Done.